packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -116,6 +116,28 @@ export class Loan {
   async listCollateralTokens (): Promise<CollateralTokensData> {
     return await this.client.call('listcollateraltokens', [], 'bignumber')
   }
+
+  /**
+   * Creates (and submits to local node and network) a token for a price feed set in collateral token.
+   *
+   * @param {SetLoanToken} loanToken
+   * @param {string} loanToken.symbol Token's symbol (unique), no longer than 8
+   * @param {string} [loanToken.name] Token's name, no longer than 128
+   * @param {string} loanToken.priceFeedId Txid of oracle feeding the price
+   * @param {boolean} [loanToken.mintable = true] Token's 'Mintable' property, default=true
+   * @param {BigNumber} [loanToken.interest = 0] Interest rate, default=0
+   * @param {UTXO[]} [utxos = []] Specific UTXOs to spend
```

Comment:
```suggestion
   * @param {UTXO[]} [utxos=[]] Specific UTXOs to spend
```

Reply:
[utxos = []] We should write like this. There are too many same examples in Jellyfish project.